companyweb_base/views/companyweb_partner_view.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="ir.ui.view" id="companyweb_partner_view">
+        <field name="name">partner.cweb_name</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Company Web">
+
+                    <field name="cweb_name_enable" invisible="1" />
+                    <field name="cweb_address_enable" invisible="1" />
+                    <field name="cweb_jur_form_enable" invisible="1" />
+                    <field name="cweb_creditLimit_enable" invisible="1" />
+                    <field name="cweb_startDate_enable" invisible="1" />
+                    <field name="cweb_endDate_enable" invisible="1" />
+                    <field name="cweb_score_enable" invisible="1" />
+                    <field name="cweb_warnings_enable" invisible="1" />
+                    <field name="cweb_url_enable" invisible="1" />
+                    <field name="cweb_vat_liable_enable" invisible="1" />
+                    <field name="cweb_balance_year_enable" invisible="1" />
+                    <field name="cweb_prefLang_enable" invisible="1" />
+
+                    <group name="cweb_group_top">
+                        <group name="cweb_name">
+                            <div /> <!-- here it is -->
+                            <field
+                                name="cweb_jur_form"
```

Comment:
Add `string` attributes for all fields in the view, because the Companyweb prefix in all labels is not helpful (we know we are in the Companyweb tab). But the Companyweb prefix is important in the orm field name so the export wizard lets the user distinguish them.